### PR TITLE
chore: bump devDependencies

### PR DESCRIPTION
**Pull request by patrickhulce**
should fix some of the security vulnerability notices

### CODE REVIEW

File: packages/server/test/server-test-suite.js [hunk 1]
```diff
@@ -376,7 +376,7 @@ function runTests(state) {
         doubleZeroBuild,
         singleZeroBuild,
       ]);
-    }, 60000);
+    }, 600e3); // this can take a really long time in CI, 00 = 1/256 chance, 0 = 1/16 chance
```
Comment (connorjclark) [inline packages/server/test/server-test-suite.js:379]: wat are you doing

Comment (patrickhulce) [reply packages/server/test/server-test-suite.js:379]: re: why 10 minutes, cuz CI is reallyyyyyy slow

re: the notation, it's a prettier [hack](https://prettier.io/playground/#N4Igxg9gdgLgprEAuEBLGAKA5AZwBYQCuANgCYAExEEA1uZBMVgDTkYCU5AvAHznAAdKOXrQcjOADoqAczZYGTdkIC+rAGwAGOAGZlUIemz4iZStTqKWbTr35CRkKOOJTZ8qBBijGWfWvItcgAqcgBGTUj2EGYQCAAHGFQxZFAAQwAnDIgAdwAFTIQcZBA04hy0gE9i2IAjDLSwGjgYAGV4xtQoGWQYDMI4WLwYAFtiAHU8dDgcDrA4VqL0VAA3dEqSsBwatGc4DJg8hpkRtOQAMzKcQZAAKxwADwAhBqaW1rSRuAAZLrgLq43e4PVpdGSuACKhC8-yQl2I11iHQy1wyJVqaVqcGIMRA8QyXRg41QpBgeGQAA5NEjstdxg14iV8TN9it-rEAI7Q+BHBLFFBpHAAWigcDgpHFuIycC5qGlRzSJzOcMBsWuI1QvX6NxwYMh3Nh8MRIBgmOJpPJSAATLE+mlUMQwQBhCAjU4lGYAVlxhGuABVMfyjTcVgMAJJQCWwVpgAmJACCkdaMEqrgBCLgKhUQA)

:)

Comment (paulirish) [reply packages/server/test/server-test-suite.js:379]: `60_000` tho

Comment (paulirish) [reply packages/server/test/server-test-suite.js:379]: damn.

image

not in node 10.

**Pull request merged**